app/payment/session_test.go

```diff
@@ -127,12 +127,13 @@ func init() {
 	chProposalNotif = perun.ChProposalNotif{
 		ProposalID: proposalID,
 		Currency:   currency.ETH,
-		Proposal: &pclient.ChannelProposal{
-			ChallengeDuration: challengeDurSecs,
-			InitBals: &pchannel.Allocation{
-				Balances: [][]*big.Int{{ownAmountRaw, peerAmountRaw}},
-			},
-		},
+		Proposal: &pclient.LedgerChannelProposal{
```

Comment:
Is it possible to use the `pclient.NewChannelProposal` function here ?

Reply:
Since it is to fill the mock data structure in the notification, keeping as '&pclient.LedgerChannelProposal' itself.